main.sh

```diff
@@ -110,7 +110,14 @@ if [[ -n "${changelog}" ]]; then
     retry curl --proto '=https' --tlsv1.2 -fsSL --retry 10 --retry-connrefused "https://github.com/taiki-e/parse-changelog/releases/download/v${parse_changelog_version}/parse-changelog-${parse_changelog_target}.tar.gz" \
         | "${tar}" xzf -
     parse_changelog_options+=("${changelog}" "${version}")
-    notes=$(./parse-changelog "${parse_changelog_options[@]}")
+
+    # If allowMissingChangelog is true then default to empty value if version not found
+    if [[ "${allowMissingChangelog}" == "true" ]]; then
```

Comment:
GitHub Actions passes input options to the script via environment variables named `INPUT_<upper_cased_option_name>`.[^1] (In this case it is probably `INPUT_ALLOWMISSINGCHANGELOG`)

[^1]: [We are assigning these to readable variable names first](https://github.com/taiki-e/create-gh-release-action/blob/0a3a16aba523e2d3ff92a744269fc789c14df703/main.sh#L30-L36), but forgot to leave a comment explaining that its code handles input options.

Since camelCase probably converts to a hard-to-read environment variable name, it would be better to use snake_case (in this case `allow_missing_changelog`) for the input option name.